README.md

```diff
@@ -328,66 +349,165 @@ The complete list of transformations supported and their usage in ImageKit can b
+| effectSharpen                 | effectSharpen(value: Int = 0)                                                                                                                                                                                                                                                                                                                        | e-sharpen               |
+| effectUSM                     | effectUSM( radius: Float, sigma: Float, amount: Float, threshold: Float)                                                                                                                                                                                                                                                                             | e-usm                   |
+| effectContrast                | effectContrast(flag: Boolean)                                                                                                                                                                                                                                                                                                                        | e-contrast              |
+| effectGray                    | effectGray(flag: Boolean)                                                                                                                                                                                                                                                                                                                            | e-grayscale             |
+| original                      | original(flag: Boolean)                                                                                                                                                                                                                                                                                                                              | orig                    |
+| Raw param string              | raw(params: String)                                                                                                                                                                                                                                                                                                                                  | -                       |
 
 </details>
 
-### File Upload
+### Responsive image loading
+To automatically set the dimensions and pixel ratio of the image , call `ImageKit.getInstance().url(...).setResponsive(...)` with a set of parameters defined below:
```

Comment:
Still missing key information regarding how responsive images will actually work and what can user expect.